applications/CompressiblePotentialFlowApplication/tests/test_CompressiblePotentialFlowApplication.py

```diff
@@ -5,7 +5,7 @@
 import run_cpp_unit_tests
 
 ##### SMALL TESTS #####
-from potential_flow_test_factory import Naca0012SmallTest
+from potential_flow_test_factory import *
```

Comment:
the previous version was what we usually do, please stick to it

Reply:
Ok!